src/App.tsx

```diff
@@ -1,8 +1,61 @@
-import React from 'react';
+import React, { useState, FC } from 'react';
 import './App.css';
+import { TodosList } from './components/TodosList/TodosList';
+import { getUsers, getTodos } from './api';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC<{}> = () => {
+  const [isLoaded, setLoaded] = useState<boolean>(false);
```

Comment:
примитивы можно не указывать, они определяются автоматически
```suggestion
  const [isLoaded, setLoaded] = useState(false);
```

Reply:
Done